components/collective-navbar/index.js

```diff
@@ -286,7 +286,7 @@ const getMainAction = (collective, callsToAction, LoggedInUser) => {
             <Settings size="1em" />
             <Span ml={2}>
               {collective.isHost && hasNewAdminPanel ? (
-                <FormattedMessage id="AdminPanel.button" defaultMessage="Admin" />
```

Comment:
Not sure whether it's a rebase issue, but this one seems back

Reply:
@Betree What is the issue with this one? I lost track.

Reply:
Similarly to https://github.com/opencollective/opencollective-frontend/pull/7241#discussion_r764591717, we should keep it as `id="AdminPanel.button"` otherwise in the navbar the admin panel button will show up as `Administrateur` in FR instead of `Admin` (or `Administration`, not sure what the current translation is)